assets/css/style.css

```diff
@@ -299,4 +299,46 @@ h1, h2, h3, h4, h5, h6 {
 .fig img {
     display: block;
     transition: opacity .1s linear;
+}
+
+/* Statistic */
+
+.statistic {
+    width: 100%;
+    height: 265px;
+    background-color: #a0dacc;
+    position: absolute;
+    bottom: 0%;  
```

Comment:
I think you don't need this. Try to add margin to create space between statistic and photos